example/res/layout/fragments_example_layout.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
+
```

Comment:
why this?

Reply:
I'll revert

Reply:
Actually I'll just leave as is since it's a no-op